self-core-impl/src/main/java/com/selfxdsd/core/GitlabWebhooks.java

```diff
@@ -118,15 +120,76 @@ public boolean add(final Project project) {
 
     @Override
     public boolean remove() {
-        throw new UnsupportedOperationException(
-            "Not yet implemented."
-        );
+        boolean removed = true;
+        for(final Webhook hook : this) {
+            if(hook.url().contains("//self-xdsd.")) {
+                LOG.debug(
+                    "Removing Self XDSD Webhook from ["
+                    + this.hooksUri + "]..."
+                );
+                final Resource response = this.resources
+                    .delete(
+                        URI.create(
+                            this.hooksUri.toString() + "/" + hook.id()
+                        ),
+                        Json.createObjectBuilder().build()
+                    );
+                final int status = response.statusCode();
+                if(status == HttpURLConnection.HTTP_NO_CONTENT) {
+                    LOG.debug("Hook removed successfully!");
+                } else {
+                    LOG.debug(
+                        "Problem while removing webhook. "
+                        + "Expected 204 NO CONTENT, but got " + status + "."
+                    );
+                    removed = removed && Boolean.FALSE;
```

Comment:
@amihaiemil  I'm just curious why: `removed = removed && Boolean.FALSE`?
can `removed && Boolean.FALSE` be != Boolean.FALSE?